content/documentation/pages/8-resources/2-faq.md

```diff
@@ -6,57 +6,6 @@ description: ''
 
 # Frequently Asked Questions
 
-## Application Starters
```

Comment:
This is really part of https://github.com/spring-io/dataflow.spring.io/issues/406